src/backend/utils/adt/lockfuncs.c

```diff
@@ -96,9 +173,23 @@ pg_lock_status(PG_FUNCTION_ARGS)
 	LockData   *lockData;
 	PredicateLockData *predLockData;
 
+	if (SRF_IS_SQUELCH_CALL())
```

Comment:
Can this be tested like in a commit https://github.com/arenadata/gpdb/commit/def226ed38062553d1f4c17daac99fc62c18c15e?